test/extended/etcd/etcd_test_runner.go

```diff
@@ -30,7 +30,7 @@ var _ = g.Describe("[sig-api-machinery] API data in etcd", func() {
 
 	oc := exutil.NewCLIWithPodSecurityLevel("etcd-storage-path", psapi.LevelBaseline).AsAdmin()
 
-	_ = g.It("should be stored at the correct location and version for all resources [Serial]", func() {
+	_ = g.It("should be stored at the correct location and version for all resources [Serial][apigroup:config.openshift.io]", func() {
```

Comment:
followup to snip this dep. *must have* for 4.12, earlier is better.  This sometime in september for this one in particular.